android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Wed Aug 24 10:01:46 CEST 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
```

Comment:
These downgrades were done due to an error in the pipeline which made me believe they were required, but were not. I then set the versions to the versions used by the most popular reactnative modules, but since that was not the issue lets keep them higher.